integration-tests/tests/failover_rclocal_crash_test.go

```diff
@@ -29,6 +29,8 @@ import (
 )
 
 func (s *failoverSuite) TestRCLocalCrash(c *check.C) {
+	c.Skip("FIXME: fails for unknown reasons")
```

Comment:
The three failover tests are passing locally for me with the changes in this branch and the latest udf [1], could you please check again?

[1] http://paste.ubuntu.com/14856762/